src/playback/internal/playbackcontroller.cpp

```diff
@@ -230,18 +232,24 @@ void PlaybackController::onSeekAction(const muse::actions::ActionData& args)
         return;
     }
 
-    if (isPlaying()) {
-        LOGD() << "Can't do seek while playing";
-        return;
-    }
-
     muse::secs_t secs = args.arg<double>(0);
-    m_lastPlaybackSeekTime = secs;
-    player()->seek(secs);
+    bool triggerPlay = args.count() > 1 ? args.arg<bool>(1) : false;
```

Comment:
I can't find where the action is dispatched with a boolean argument, am i missing anything?